photutils/psf/photometry.py

```diff
@@ -289,20 +297,21 @@ class for more information.
 
     def __init__(self, psf_model, fit_shape, *, finder=None, grouper=None,
                  fitter=LevMarLSQFitter(), fitter_maxiters=100,
-                 localbkg_estimator=None, aperture_radius=None,
+                 xy_bounds=None, localbkg_estimator=None, aperture_radius=None,
                  progress_bar=False):
 
         self._param_maps = self._define_param_maps(psf_model)
         self.psf_model = _validate_psf_model(psf_model)
 
-        self.fit_shape = as_pair('fit_shape', fit_shape, lower_bound=(0, 1),
+        self.fit_shape = as_pair('fit_shape', fit_shape, lower_bound=(1, 0),
```

Comment:
Is this change intentional?  I'm not sure what it does, but it seems smuggled in

Reply:
It's a tiny "fix" that I noticed during testing.  All this change does is change the error message if one were to input `fit_shape = 0`.  It could be a separate PR, but it's tiny.....